src/api/api.js

```diff
@@ -0,0 +1,23 @@
+const API_URL = `https://mate-api.herokuapp.com`;
+
+export const getTodos = () => (
+  fetch(`${API_URL}/todos`)
+    .then((response) => {
+      if (response.ok) {
+        return response.json();
+      }
+
+      throw new Error(`${response.status}: ${response.statusText}`);
+    })
+);
+
+export const getUser = id => (
+  fetch(`${API_URL}/users/${id}`)
+    .then((response) => {
+      if (response.ok) {
+        return response.json();
+      }
+
+      throw new Error(`${response.status}: ${response.statusText}`);
+    })
+);
```

Comment:
Here you have almost identical functions.  You can create 1 function, which will get a response, and then just use it in `getTodos` and `getUsers` with different params